lib/hyacinth/adapters/external_identifier_adapter/datacite.rb

```diff
@@ -4,109 +4,136 @@ module Hyacinth
   module Adapters
     module ExternalIdentifierAdapter
       class Datacite < Abstract
-        IDENTIFIER_STATUS = { public: 'public',
-                              reserved: 'reserved',
-                              unavailable: 'unavailable' }.freeze
+        DOI_STATES = [:draft, :findable, :registered].freeze
```

Comment:
This `DOI_STATES = [:draft, :findable, :registered].freeze` line appears in three different files, so it would be good to de-duplicate and reference only one from all three places so they can't get out of sync.

Reply:
And actually, I just checked and I can't find any usage references to `DOI_STATES` anywhere else in the code.  Is it possible that all three copies of this constant are unused, or did I miss something?

Reply:
DOI_STATES removed from code.